content/articles/2019-02-bigrquery-1-1-0.Rmarkdown

```diff
@@ -0,0 +1,40 @@
+---
+title: bigrquery 1.1.0
+author: Mara Averick
+date: '2019-02-15'
+slug: bigrquery-1-1-0
+description: > 
+  bigrquery 1.1.0 is now on CRAN.
+categories:
+  - package
+photo:
+  url: https://www.pexels.com/photo/green-and-red-abstract-painting-1070534/
+  author: Steve Johnson
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>"
+)
+```
+
+We're delighted to announce that [bigrquery](https://bigrquery.r-dbi.org/) 1.1.0 is now on CRAN. bigrquery makes it easy to work with data stored in [Google BigQuery](https://developers.google.com/bigquery/), a hosted database for big data. This is a minor release, with some improved type support, and SQL translation. For a complete list of updates, please see the [Changelog](https://bigrquery.r-dbi.org/news/index.html).
```

Comment:
Should this link to a specific version? (https://bigrquery.r-dbi.org/news/index.html#bigrquery-1-1-0)